packages/bootstrap/package.json

```diff
@@ -19,14 +19,19 @@
     "access": "public"
   },
   "scripts": {
+    "build": "npm run sass",
+    "watch": "npm run sass:watch",
+    "swatches": "npm run sass:swatches",
+    "sass": "gulp sass",
+    "sass:watch": "gulp sass:watch",
+    "sass:swatches": "gulp sass:swatches",
+    "dart": "gulp dart",
+    "dart:watch": "gulp dart:watch",
+    "dart:swatches": "gulp dart:swatches",
     "lint": "gulp lint",
     "api": "gulp api",
     "api-check": "npm run api && git diff --exit-code --quiet -- docs/ || (echo -e '\\033[0;31mERROR: API docs are out of date' && exit 1)",
-    "build": "gulp sass",
-    "dart": "gulp dart",
-    "watch": "gulp sass:watch",
     "embed-assets": "gulp assets",
-    "swatches": "gulp swatches",
     "test": "npm run lint && npm run build && npm run api-check",
     "prepublishOnly": "bash ./build/embed-dependencies 'bootstrap' '@progress/kendo-theme-default' && npm run build && npm run swatches",
```

Comment:
I'd use a gulp task that lists "sass" and "sass:swatches" as dependencies instead of calling them indirectly.

Reply:
agreed